src/main/java/com/mission2/racingcar/CarRacingService.java

```diff
@@ -0,0 +1,111 @@
+package com.mission2.racingcar;
+
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class CarRacingService {
+    public static final int MAX_CAR_COUNT = 5;
+    public static final int MAX_GAME_COUNT = 10;
+    public static final int INIT_SCORE = 1;
+    public static final int FORWARD = 4;
+    public static final String CAR_NAME_DELIMITER = ",";
+
+    public void game(String[] carNames, int gameCount) {
+        System.out.println("실행결과");
+        Race race = initRace(carNames, gameCount);
+        proceedGame(race);
+        printWinner(getWinners(race.getCars()));
+    }
+
+    /**
+     * 자동차 이름 문자열 길이를 확인한다.
+     */
+    public boolean checkCarNames(String[] carNames) {
+        long count = Arrays.stream(carNames).filter(car -> car.length() > MAX_CAR_COUNT).count();
+
+        return (count > 0) ? false : true;
```

Comment:
```suggestion
        return count <= 0;
```

요런 표현들이 많이 보이는데요, boolean return에서는 항상 simplify 를 고려하면 좋을 것 같아요 ㅎㅎ 다른 곳도 같이 확인해 주세요 :)